views/components/App.jsx

```diff
@@ -49,52 +55,96 @@ export default class App extends React.Component {
     });
   }
 
+  selectInput (e) {
+    var input = WebMidi.getInputById(e.target.value);
+    input.addListener('controlchange', "all", this.midiRxCC)
+    this.setState({
+      currentInput: input,
+    });
+  }
+
+  midiRxCC (e) {
+    console.log("RXed " + e);
+    // change Rx to true to cause LED to blink
+    this.setState({inputRx: true});
+  }
```

Comment:
Technically, you should also implement a `componentWillUnmount` method that also clears the timeout just like above.  React yells at you when you try to modify state on an unmounted component.  However, if this component is never gonna be unmounted, you probably can skip this step.